### DISCUSSION TITLE
서버로부터 error 메세지를 받았을 때 error.status의 타입

### DISCUSSION THREAD

**Discussion by yousunzoo** [KDT4-team6/Paldo#29] [General]
## 코드 예시
```javascript
export async function requestSignup(signupData) {
  const res = await fetch(`${url}auth/signup`, {
    method: "POST",
    headers,
    body: JSON.stringify(signupData),
  })
    .then((response) => {
      if (!response.ok) {
        throw new Error(response.status);
      }
      return response.json();
    })
    .then((result) => {
      setUserInfo(result, signupData.email);
      history.back();
    })
    .catch((error) => {
      console.log(error, typeof error);
    });
}
```

![image](https://user-images.githubusercontent.com/102499959/218308597-e951ead7-3827-4a34-b7f5-f9f6e325b202.png)

* `response.status`는 `Error: 401`이라는 값을 반환하길래 `error === "Error: 401`이라는 코드를 사용해서 alert 창을 구현하려고 하니 에러가 발생했습니다.
* 그래서 `response.status`의 type을 확인해봤더니 object였습니다.
* 혹시 에러 상태를 가지고 alert 창 같은 것을 구현하실 떄 일치 연산자가 아닌 동등 연산자를 사용하시면 될 것 같습니다.

**Comment by bongsee** [comment]
사례 공유 감사합니다~!
